src/swap/SwapReviewScreen.test.tsx

```diff
@@ -129,4 +132,37 @@ describe('SwapReviewScreen', () => {
       )
     })
   })
+
+  it('should have correct analytics on screen open', () => {
+    const mockBuyAmount = '3000000000000000000'
+    const mockSellAmount = '1000000000000000000'
+    mockFetch.mockResponse(
+      JSON.stringify({
+        unvalidatedSwapTransaction: {
+          sellToken: mockCeloAddress,
+          buyToken: mockCusdAddress,
+          buyAmount: mockBuyAmount,
+          sellAmount: mockSellAmount,
+          price: '3.00',
+          gas: '300000',
+          gasPrice: '500000000',
+        },
+      })
+    )
+
+    render(
+      <Provider store={store}>
+        <SwapReviewScreen />
+      </Provider>
+    )
+
+    expect(ValoraAnalytics.track).toHaveBeenCalledWith(SwapEvents.swap_review_screen_open, {
+      buyAmount: mockBuyAmount,
+      fromToken: mockCeloAddress,
+      toToken: mockCusdAddress,
+    })
+  })
+
+  it.todo('should correctly dispatch swapStart')
```

Comment:
todo?

Reply:
Added in a3e5f498c346061c52a24d7ec3eef1758809aeed!